project.clj

```diff
@@ -10,18 +10,23 @@
   :dependencies [[org.clojure/clojure "1.7.0"]
 
                  ;; begin version conflict resolution dependencies
-                 [clj-time "0.10.0"]
+                 [clj-time "0.11.0"]
                  [commons-codec "1.9"]
                  [org.clojure/tools.macro "0.1.5"]
+                 [org.clojure/tools.reader "1.0.0-alpha1"]
```

Comment:
It would be nice to avoid using an `alpha` version here if possible.

Would `0.10.0` work instead?  Failing that, how about `1.0.0-beta1`?